server/src/controllers/eventController.ts

```diff
@@ -116,37 +117,37 @@ eventController.post('/checkout', async (req: Request, res: Response) => {
 
     if (orderSubTotal + feeTotal - discountAmount > .49) {
       toSend = await createStripeCheckoutSession(
-        contactid,
-        formData.email,
-        cartRows,
-        {...discount, amountOff: discountAmount},
+          updatedContact,
+          ticketCartRows.concat((donationCartRow? [donationCartRow]: []).concat(subscriptionCartRows)),
```

Comment:
I think the `ticketCartRows.concat(...` is reverting a change from the last PR that merged. I think it is supposed to still be `cartRows`